fixes #32: adding getting involved info to README.md (#35)
beman-project/beman: README.md

This project is organized by our [Governance structure](docs/governance.md).

### Participating and contributing

> We welcome anyone, who wants to participate in the Beman project either through discussions on
> [Discourse](https://discourse.boost.org/t/welcome-to-beman-project-development/3), or through
> contributions on GitHub. If you are interested, please join our discourse community to stay up to
> date with the most recent developments.

Reviewer: ```suggestion
We welcome all to participate in the Beman project through 
[Discourse](https://discourse.boost.org/t/welcome-to-beman-project-development/3) discussions or GitHub contributions. Our discourse community is the best way to stay up to
date with the most recent developments.
```

Author: thanks, updated!

> If you are interested to contribute on GitHub, please take a look at the open issues of the
> different [repositories](https://github.com/orgs/beman-project/repositories) of the Beman
> organization. Issues that are marked as `good first issue` are a good starting point for new
> contributors. We try to have an assigned mentor for each of these issues to give guidance to new
> contributors. Don't hesitate to post on discourse, if you have any questions.

Reviewer: ```suggestion
If you are looking for ways to contribute code, see the open issues of [beman-project repositories](https://github.com/orgs/beman-project/repositories). The [beman repository's issue list](https://github.com/beman-project/beman/issues), in particular, has implementation requests for various papers. Issues marked with `good first issue` are perfect for new
contributors and usually have an assigned mentor. Don't hesitate to post on discourse with any questions.
```

Author: updated
